client/analytics/report/orders/index.js

```diff
@@ -300,8 +312,24 @@ export default compose(
 			},
 			select
 		);
+
+		const { getOrders, isGetOrdersError, isGetOrdersRequesting } = select( 'wc-admin' );
+		const tableQuery = {
+			orderby: query.orderby || 'date',
+			order: query.order || 'asc',
+			page: query.page || 1,
+			per_page: query.per_page || 25,
+			after: datesFromQuery.primary.after + 'T00:00:00+00:00',
```

Comment:
Since we are appending these timestamps in multiple areas of the code, do you think we should create some simple util methods to do this? That way if we have to wholesale change it down the road it isn't as bad, and we don't have to keep typing in start/end of day times? Obviously this could be done as a follow-up, just curious to hear your thoughts?

Reply:
Absolutely, I think it's a good idea!

Reply:
PR created: #542.